components/hospital-detail/ContactCard.tsx

```diff
@@ -26,8 +26,8 @@ const ContactCard: React.FC<ContactProps> = ({ website, phoneNumber }) => {
         <h3 className="text-lg pl-1">Web</h3>
         <div className="flex flex-wrap p-2 pl-0">
         <svg className="flex flex-wrap justify-center self-center" width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
is there a react icon we can use in place of this svg, it would be more clear if react icon is used.